packages/gatsby-transformer-react-docgen/src/on-node-create.js

```diff
@@ -10,6 +10,19 @@ const digest = str =>
 const propsId = (parentId, name) => `${parentId}--ComponentProp-${name}`
 const descId = parentId => `${parentId}--ComponentDescription`
 
+function canParse(node) {
+  return (
+    node &&
```

Comment:
I don't think this check is really needed!

```suggestion
```

Reply:
ya it more out of an abundance of caution i can remove if you like?

Reply:
It's probably OK. Let's just merge this, thank you!